### PR TITLE
LPS-118829 Remove AUI from DM (part 1)

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
I divided the main task [LPS-118829 Remove AUI from main.js in Documents&Media ](https://issues.liferay.com/browse/LPS-118829 )in two. This replaces the AUI component, and it only misses the upload related functionality. 

functions migrated from main.js: 
- [x] _moveSingleElement
- [x] _openDocument: not used 
- [x] _processAction: not used. Inline code in __moveToTrash_
- [x] _handleSearchContainerRowToggled - simplified, _selectedFileEntries_ was not used
- [x] _moveCurrentSelection
- [x] _moveToFolder
- [x] _moveToTrash
- [x] showFolderDialog

Missing functions that will be covered in subtask [LPS-155521](https://issues.liferay.com/browse/LPS-155521)
- [ ] _plugUpload 
- [ ] getFolderId
- [ ] initializations in initializer function

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/DocumentLibrary.js [hunk 1]
```diff
@@ -12,10 +12,18 @@
  * details.
  */
 
+const TPL_MOVE_FORM =
```
Comment (boton) [inline modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/DocumentLibrary.js:15]: Fortunately, this TPL is only about string replacement, we can update this TPL to a template literal inlining it or converting it into a function.

**Review by boton — COMMENTED** [review]
I leave a couple of inline comments

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
ci:reevaluate:1297401_3231

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/DocumentLibrary.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+		const newForm = document.createElement('div');
+
+		newForm.appendChild(
+			buildFragment(
+				`<form action="${editEntryUrl}" class="hide" method="POST"><input name="${namespace}cmd" value="move"/>` +
+					`<input name="${namespace}newFolderId" value="${newFolderId}"/>` +
+					`<input name="${namespace}${parameterName}" value="${parameterValue}"/>` +
+					`<input name="${namespace}redirect" value="${redirectUrl}"/>` +
+					`</form>`
```
Comment (boton) [inline modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/DocumentLibrary.js:67]: The line breaks and concatenation isn´t needed you can use somethings like
```js
`<form action="${editEntryUrl}" class="hide" method="POST"><input name="${namespace}cmd" value="move"/>
	<input name="${namespace}newFolderId" value="${newFolderId}"/>
	<input name="${namespace}${parameterName}" value="${parameterValue}"/>
	<input name="${namespace}redirect" value="${redirectUrl}"/>
</form>`
```

**Review by boton — APPROVED** [review]
I leave a small inline comment but LGTM

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/DocumentLibrary.js (hunk 2 above)
Comment (ambrinchaudhary) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/DocumentLibrary.js]: 🙈 I can't believe I leave it that way 😄 . Fixed and thanks for reviewing!

**Comment by ambrinchaudhary** [comment]
ci: forward

**Pull request closed without merging**